src/Lower.cpp

```diff
@@ -424,6 +427,24 @@ void lower_impl(const vector<Function> &output_funcs,
         debug(1) << "Skipping GPU offload...\n";
     }
 
+    std::vector<LoweredFunc> closure_implementations;
+    debug(1) << "Lowering Parallel Tasks...\n";
+    s = lower_parallel_tasks(s, closure_implementations, pipeline_name, t);
+    // Process any LoweredFunctions added by other passes. In practice, this
```

Comment:
The "In practice, ..." comment makes me wonder if this should be an error condition instead. Does it at least warrant a TODO and an Issue?